discovery-infra/test_infra/helper_classes/kube_helpers/installenv.py

```diff
@@ -0,0 +1,232 @@
+import waiting
+import yaml
+
+from typing import Optional, Union, Dict
+from pprint import pformat
+
+from kubernetes.client import ApiClient, CustomObjectsApi
+from kubernetes.client.rest import ApiException
+
+from tests.conftest import env_variables
+
+from .globals import *
+from .base_resource import BaseCustomResource
+from .secret import deploy_default_secret, Secret
+from .common import logger
+
+
+class InstallEnv(BaseCustomResource):
+    """
+    InstallEnv is used to generate cluster iso.
+    Image is automatically generated on CRD deployment, after InstallEnv is
+    reconciled. Image download url will be exposed in the status.
+    """
+    _api_group = 'adi.io.my.domain'
+    _api_version = 'v1alpha1'
```

Comment:
let's be consistent with ``_version`` as in ``ClusterDeployment``

Reply:
cluster deployment CRD is part of hive definitions where it is v1
installenv CRD made by us, where we still don't have v1 for our api

Reply:
Same as above ^